package.json

```diff
@@ -3,31 +3,13 @@
   "version": "0.3.0",
```

Comment:
We are currently at version 0.3.1 so we need to bump this to 0.3.2 for example. Then please run `npm i` so the package-lock.json is also updated. Probably need to do the same for all packages individually.

Reply:
by switching to pnpm we no longer have a package-lock